### ISSUE TITLE
fix display for topic-poll

### ISSUE THREAD

**Issue by Elblinator** [OpenSlides/openslides-client#2412]
after creating a new poll in topic '100% base:' is displayed 

The '100% base:' should not be displayed.

![bug-100Base-vote](https://github.com/OpenSlides/openslides-client/assets/69210919/54eedd44-c969-4a2a-af39-112f8cf6ede6)

![create-topic-poll](https://github.com/OpenSlides/openslides-client/assets/69210919/4143db8f-1f16-449c-9750-3ae6a586be2b)

**Comment by Elblinator** [comment]
> Which may seem strange until you consider that 100% for option 4 with an additional 20% for option 3 just means that everyone agreed with "Fries with ketchup", while only 20% of the delegates liked "vegetable soup" I wouldn't want to destroy this functionality by "fixing" it

I agree, the percentages are correct. The pie chart (if the options in sum are bigger than 100%) is misleading. 
One pie chart represent 100%, having more than 100% should **not** be represented by one pie chart. 
This is why the pie chart should not be displayed if the max percentage is bigger 100.

**Comment by MSoeb** [comment]
Note: 100% base should be integrated.

**Comment by reiterl** [comment]
It doen't seem to be "high" as nothing has happend for a long time. Remove high.

**Comment by rrenkert** [comment]
We will have a new vote-service including a client-side rework, unifying the voting.

**Comment by bastianjoel** [comment]
done in #6250
